Gems/Atom/Feature/Common/Assets/Shaders/Materials/MaterialInputs/SilhouetteInput.azsli

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+option enum class SilhouetteType { None, Silhouette, Outline } o_silhouetteType = SilhouetteType::Silhouette;
+option enum class HighlightType { None, Silhouette, SilhouetteOnTop, Outline, OutlineOnTop, SilhouetteOutlined, SilhouetteOutlinedOnTop } o_highlightType = HighlightType::None;
```

Comment:
I do not understand the meaning of SilhouetteType. HighlightType defines highlight, but what is SilhouetteType for?

Reply:
I pulled these in based on what I'm seeing in `HighlightComponent.cpp` It has a `Type` field and a `HighlightType` field, so I named the first one `SilhouetteType`, but I'm not really sure how it was used because I don't have the original shader code.

Reply:
I've removed these options